core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -200,11 +150,23 @@ abstract class EndToEndAuthorizationTest extends IntegrationTestHarness with Sas
     */
   @BeforeEach
   override def setUp(testInfo: TestInfo): Unit = {
+
+    // The next two configuration parameters enable ZooKeeper secure ACLs
+    // and sets the Kafka authorizer, both necessary to enable security.
+    this.serverConfig.setProperty(KafkaConfig.ZkEnableSecureAclsProp, "true")
+    this.serverConfig.setProperty(KafkaConfig.AuthorizerClassNameProp, authorizerClass.getName)
+
+    // Set the specific principal that can update ACLs.
+    this.serverConfig.setProperty(AclAuthorizer.SuperUsersProp, kafkaPrincipal.toString)
+
     super.setUp(testInfo)
+
+    val superuserAdminClient = createSuperuserAdminClient()
```

Comment:
Do we need to create `AclClusterAction, AclAlter` acls? since kafkaPrincipal is super user, we dont need these acls.

Reply:
I don't know that we do. I know that it works with it, but I haven't tried testing without it.

Reply:
I tested as we discussed and in fact it is not necessary. I removed the code at setUp().